test/tree/catkin/multi_dep_type_catkin_package/package.xml

```diff
@@ -0,0 +1,19 @@
+<package format="2">
```

Comment:
```suggestion
<package format="3">
```
Maybe nit, and open to counter suggestion; my personal preference is new development to  happen in format 3.

Reply:
All the dependency types considered in the new feature were introduced in format 2. It is the minimum format required for this feature. Format 3 is compatible with it so it should work too (we could change it without breaking the tests). However, it is important to highlight that I'm ignoring the new dependency type `group_depend`, introduced in format 3. I was not sure how I would handle it, so I excluded it from this update.

If the current behavior is considered adequate for format 3 given the `group_depend` ignoring, I'm ok with changing it.